user-access-internal-model/src/test/java/no/unit/nva/useraccessmanagement/dao/UserDbTest.java

```diff
@@ -132,14 +141,6 @@ void copyShouldReturnBuilderWithFilledInFields() throws InvalidEntryInternalExce
         assertThat(copy, is(not(sameInstance(originalUser))));
     }
 
-    @Test
-    void setPrimaryHashKeyThrowsExceptionWhenKeyDoesNotStartWithType() {
```

Comment:
test is not valid any more because keys are never set anymore. They are only calculated